pkgs/top-level/all-packages.nix

```diff
@@ -16549,4 +16549,8 @@ tweakAlias = _n: alias: with lib;
     removeAttrs alias ["recurseForDerivations"]
   else alias;
 
-in lib.mapAttrs tweakAlias aliases // self; in pkgs
+in
+  lib.mapAttrs tweakAlias aliases // self;
+
```

Comment:
Because the lines above are the end of a function, while the 2 lines below are the end of an enclosing let-in which goes through the whole file.

As soon as all packages are moved in another file than the fix-point logic of nixpkgs, I will re-indent these lines properly.  In the mean time this extra line is a way to split the 2.